### PR TITLE
976 convert start pages to generate markdown

### PR DESCRIPTION
**Pull request by anagradova**
convert_markdown_template in formatters.py accepts a markdown file name string, will read the ".md" markdown file from the 'app/content' directory and return an HTML string for rendering. The md file can contain markdown, HTML, and jinja2 tags. The processing pipeline renders markdown first and then tags.

### CODE REVIEW

File: app/content/get-started.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <p>You can use the Notify API to send messages automatically.</p>
+    <p>Our <a class="usa-link" href="{{ url_for('main.documentation') }}">documentation</a> explains how to integrate the API with a web application or back office system.</p>
+  </li> -->
+
+  <li class="usa-process-list__item padding-bottom-4">
+    <h2 class="usa-process-list__heading line-height-sans-3">Start sending messages</h2>
+    {% if not current_user.is_authenticated or not current_service %}
+    <p>When you’re ready to send messages to people outside your team, go to the <b class="bold">Settings</b> page and select <b class="bold">Request to go live</b>. We’ll approve your request within one working day.</p>
+    {% else %}
+    <p>You should <a class="usa-link" href="{{ url_for('.request_to_go_live', service_id=current_service.id) }}">request to go live</a> when you’re ready to send messages to people outside your team. We’ll approve your request within one working day.</p>
```
Comment (stvnrlly) [inline app/content/get-started.md:45]:
```suggestion
    <p>You should <a class="usa-link" href="{{ url_for('.support') }}">request to go live</a> when you’re ready to send messages to people outside your team. We’ll approve your request within one working day.</p>
```

I recently changed https://github.com/GSA/notifications-admin/blob/main/app/templates/views/get-started.html, so the `request_to_go_live` route no longer exists. This should fix the current build error.

File: app/content/get-started.md
Comment (stvnrlly) [inline app/content/get-started.md:1]: HTML is valid Markdown, but for our purposes we'd like to have this file be non-HTML Markdown. That'll hopefully make it easier for non-developers to edit this.

For example, the first line here should be 
```
# Get started
```

**Review by stvnrlly — CHANGES_REQUESTED** [review]
The Python part looks like it should do exactly what we're looking for 👍🏻 . I left some feedback about the Markdown parts.

File: app/content/get-started.md
Comment (anagradova) [reply app/content/get-started.md:1]: Would you like me to modify the md file to only use markdown (where possible)? I assume we'll keep the jinja template tags in the markdown file?

Comment (stvnrlly) [reply app/content/get-started.md:1]: Yes, please! I am assuming that Jinja tags will still work, but I have not tried it myself to confirm that.

Comment (anagradova) [reply app/content/get-started.md:1]: They will work. It passes through the markdown conversion first so the template will look normal (tags + HTML) to the render_template_string method.

**Comment by ccostino** [comment]
@anagradova I think we're good to go here!  We just need one more update against `main` as a few other dependency updates had gone in and I want to make sure the Poetry files are in good shape. :-) Thanks!

**Review by ccostino — APPROVED** [review]
Thanks, @anagradova!

**Review by stvnrlly — APPROVED** [review]

**Pull request merged**